platform-tests/src/test/java/org/junit/platform/launcher/core/LauncherFactoryTests.java

```diff
@@ -22,15 +22,36 @@
 import org.junit.platform.commons.util.ReflectionUtils;
 import org.junit.platform.launcher.Launcher;
 import org.junit.platform.launcher.LauncherDiscoveryRequest;
+import org.junit.platform.launcher.TestExecutionListener;
 import org.junit.platform.launcher.TestIdentifier;
 import org.junit.platform.launcher.TestPlan;
+import org.junit.platform.launcher.listener.CustomTestExecutionListener;
 
 /**
  * @since 1.0
  */
 class LauncherFactoryTests {
 
 	@Test
+	void testCustomTestExecutionListenerServiceIsLoaded() {
+		DefaultLauncher launcher = (DefaultLauncher) LauncherFactory.create();
+		TestExecutionListenerRegistry registry = launcher.getTestExecutionListenerRegistry();
+		List<TestExecutionListener> listeners = registry.getTestExecutionListeners();
+		assertThat(listeners).hasSize(2);
+		List<TestExecutionListener> one = listeners.stream().filter(
+			listener -> listener instanceof CustomTestExecutionListener).collect(toList());
+		assertThat(one).hasSize(1);
+		CustomTestExecutionListener listener = (CustomTestExecutionListener) one.get(0);
+		assertThat(listener.testPlanExecutionStarted).isEqualTo(0);
```

Comment:
Asserting that each of these counts is zero is akin to asserting that a getter/setter method pair in a POJO work.

In other words, I don't see any added value in these assertions. However, if you were to assert the counters after an actual test run, that would be different. But I don't think the latter is truly necessary since we have other integration tests in place for such behavior in other TELs.
 
Now that I think about it, we don't really even need two different types of concrete TELs: they could both be a no-op TEL.